assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
+            "name": "Vally Two",
             "rwy": {
-                "8L" : ["_R26R-1DME", "HINKU", "VAPOR", "SHALW"],
-                "8R" : ["_R26L-1DME", "GOZZO", "VAPOR", "SHALW"],
-                "12" : ["_R30-1DME", "RIKEE", "VAPOR", "SHALW"],
-                "9"  : ["_R27-1DME", "RIKEE", "VAPOR", "SHALW"],
-                "26L": ["_R8R-1DME", "METTS", "HAUSR", "FORKK", "SHALW"],
-                "26R": ["_R8L-1DME", "METTS", "HAUSR", "FORKK", "SHALW"],
-                "30" : ["_R12-1DME", "METTS", "HAUSR", "FORKK", "SHALW"],
-                "27" : ["_R9-1DME", "METTS", "HAUSR", "FORKK", "SHALW"]
+                "KMIA8L": ["HINKU", "VAPOR"],
+                "KMIA8R": ["GOZZO", "VAPOR"],
+                "KMIA9": ["RIKEE", "VAPOR"],
+                "KMIA12": ["RIKEE", "VAPOR"],
+                "KMIA26L": ["METTS", "HAUSR", "FORKK"],
+                "KMIA26R": ["METTS", "HAUSR", "FORKK"],
+                "KMIA27": ["METTS", "HAUSR", "FORKK"],
+                "KMIA30": ["METTS", "HAUSR", "FORKK"]
             },
+            "body": ["SHALW", "JANUS"],
             "exitPoints": {
-                "VALLY": ["JANUS", "VALLY"]
+                "VALLY": ["VALLY"]
             },
-            "draw": [["JANUS","VALLY*"]]
+            "draw": [
+                ["HINKU", "VAPOR", "SHALW", "JANUS", "VALLY*"],
+                ["GOZZO", "VAPOR"],
+                ["RIKEE", "VAPOR"],
+                ["METTS", "HAUSR", "FORKK", "SHALW"]
+            ]
         },
         "WINCO2": {
             "icao": "WINCO2",
             "name": "Winco Two",
             "rwy": {
-                "8L" : ["_R26R-1DME", "SENOY", "CIMBA", "FORKK", "BORLE"],
-                "8R" : ["_R26L-1DME", "SENOY", "CIMBA", "FORKK", "BORLE"],
-                "12" : ["_R30-1DME", "SENOY", "CIMBA", "FORKK", "BORLE"],
-                "9"  : ["_R27-1DME", "SENOY", "CIMBA", "FORKK", "BORLE"],
-                "26L": ["_R8R-1DME", "OHOFO", "BORLE"],
-                "26R": ["_R8L-1DME", "OHOFO", "BORLE"],
-                "30" : ["_R12-1DME", "OHOFO", "BORLE"],
-                "27" : ["_R9-1DME", "OHOFO", "BORLE"]
+                "KMIA8L": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA8R": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA9": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA12": ["SENOY", "CIMBA", "FORKK"],
+                "KMIA26L": ["OHOFO"],
+                "KMIA26R": ["OHOFO"],
+                "KMIA27": ["OHOFO"],
+                "KMIA30": ["OHOFO"]
             },
+            "body": ["BORLE", "MEAGN"],
             "exitPoints": {
-                "WINCO": ["MEAGN", "WINCO"]
+                "WINCO": ["WINCO"]
             },
-            "draw": [["FORKK", "BORLE"], ["OHOFO", "BORLE", "MEAGN", "WINCO*"]]
+            "draw": [
+                ["SENOY", "CIMBA", "FORKK", "BORLE", "MEAGN", "WINCO*"],
+                ["OHOFO", "BORLE"]
+            ]
         }
     },
     "stars": {
         "ANNEY3": {
```

Comment:
ANNEY3 is now ANNEY4

Reply:
reverified and updated all SIDs/STARs.

done